### ISSUE TITLE
Bug (?) in First interactive lighthouse audit

### ISSUE THREAD

**Issue by PandaSandStudio** [GoogleChrome/lighthouse#4145]
I was testing my lightweight preact app with lighthouse, under build circumstances. I was surprised to see that although the app first paint thing was around 2000ms, the first interactive time was around 22000ms, even though the phone displayed was fully interactive. After a bit of debugging, I realised that it was the "typewriter" component, which constantly was changing its text and re-rendering. So I removed the component from the display and the first interactive time went down to 2000ms. Please fix !!

**Comment by PandaSandStudio** [comment]
I think the component was tricking lighthouse into thinking that the page wasnt fully interactive even though it was (because javascript was constantly running and changing the html)

**Comment by wardpeet** [comment]
@PandaSandStudio thanks for filling this but but we have this issue with other components like carousels and videos that start autoplaying in the viewport.

It's something we can't fix at the moment

**Comment by PandaSandStudio** [comment]
@patrickhulce I disagree. The javascript execution is just an animation which enhances the experience - and the user can still interact perfectly well
